l10n/en/banners/fundraising.ftl

```diff
@@ -0,0 +1,14 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+banner-fundraising-title = The future is being decided online. Help { -brand-name-mozilla } fight for a web for all of us.
```

Comment:
Nit insert the URL of the home page here so localizers have a link to preview their work?